File: Que_03.py
# Count Even and Odd Numbers in a List
# Input: [1, 2, 3, 4, 5, 6] → Output: Even=3, Odd=3


def count_even_and_odd(arr):
    even = 0
    odd = 0
    for n in arr:
        if n % 2 == 0:
            even += 1
        else:
            odd += 1
    print(f"Even={even}, odd={odd}")

count_even_and_odd([1, 2, 3, 4, 5, 6])

File: Que_03.js
// Count Even and Odd Numbers in a List
// Input: [1, 2, 3, 4, 5, 6] → Output: Even=3, Odd=3

function countEvenOdd(arr){
    let even = 0
    let odd = 0

    for (let i = 0; i < arr.length; i++) {
        if (i % 2 == 0) {
            even += 1
        }
        else {
            odd += 1
        }   
    }
    console.log(`Even= ${even}, Odd= ${odd}`)
}

countEvenOdd([1, 2, 3, 4, 5, 6])